cf/developer-access-group.yaml

```
@@ -182,6 +182,16 @@ Resources:
                 Resource:
                   - "arn:aws:s3:::cf-templates-*"
 
+        - PolicyName: "can-delete-from-staging-buckets"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - 's3:DeleteObject'
+                Resource:
+                  - "arn:aws:s3:::*staging*/*"
```

I would do something more precise like
```suggestion
                  - ""arn:aws:s3:::*-staging-${AWS::AccountId}/*""
```